Docs/HowFuzzilliWorks.md

```diff
@@ -766,29 +766,29 @@ As previously discussed, there needs to be some guidance mechanism to direct the
 An example for a ProgramTemplate is shown next.
 
 ```swift
-ProgramTemplate("JITFunction") { b in
+fileprivate let JITFunction = ProgramTemplate("JITFunction") { b in
```

Comment:
Nit: this doesn't really need the `fileprivate let JITFunction =` part, but if you prefer adding that maybe make it `let JITFunctionTemplate =` or so so it's consistent with the templates e.g. here https://github.com/googleprojectzero/fuzzilli/blob/main/Sources/FuzzilliCli/Profiles/V8Profile.swift